api/src/main/proto/stellarstation/api/v1/radio/radio.proto

```diff
@@ -60,6 +63,27 @@ message CommunicationProtocol {
   }
 }
 
+// The type of polarization the antenna will use.
+enum Polarization {
+  // Any polarization can be used for communications.
+  ANY = 0;
+
+  // Horizontal polarization.
+  HORIZONTAL = 1;
+
+  // Horizontal polarization.
+  VERTICAL = 2;
+
+  // Left hand circular polarization.
+  LEFT_HAND_CIRCULAR = 3;
+
+  // Right hand circular polarization.
+  RIGHT_HAND_CIRCULAR = 4;
+
+  // Simultaneous left and right hand circular polarization.
+  SIMULTANEOUS_LEFT_HAND_RIGHT_HAND_CIRCULAR = 5;
```

Comment:
I don't think this is a thing...

Reply:
Thanks, I think I got confused looking at some documentation. Removed!